modules/fbs/includes/fbs.reservation.inc

```diff
@@ -389,18 +389,23 @@ function fbs_reservation_options_submit($account, $values) {
  * Tries to find the users default interest period based on the users profile2
  * provider profile.
  */
- function fbs_reservation_default_interest_period($account = NULL) {
+function fbs_reservation_default_interest_period($account = NULL) {
   $account = is_null($account) ? $GLOBALS['user'] : $account;
   $profile = ding_user_provider_profile($account);
```

Comment:
It would be nice to make it explicit in [the DocBlock for `ding_user_provider_profile()`](https://github.com/ding2/ding2/blob/e55599a6860bdab0783904abb926af07973108fe/modules/ding_user/ding_user.module#L578-L579) that it may return `NULL`.

Reply:
I added this in 858daca.